projects/storefrontlib/src/lib/store-finder/components/store-finder-search-result/store-finder-search-result.component.ts

```diff
@@ -61,15 +56,17 @@ export class StoreFinderSearchResultComponent implements OnInit, OnDestroy {
       this.searchQuery.useMyLocation
     );
 
-    this.isLoading$ = this.store.pipe(select(fromStore.getStoresLoading));
-    this.locations$ = this.store.pipe(select(fromStore.getFindStoresEntities));
+    this.isLoading$ = this.storeFinderService.getStoresLoading();
+    this.locations$ = this.storeFinderService.getFindStoresEntities();
     this.ngUnsubscribe = this.locations$
```

Comment:
`initialize` is inside `this.route.queryParams.subscribe`. So that means there is `subscribe` inside another `subscribe`. Normally we don't do that. And, any `subscribe` in `ngOnInit` need to be `unsubscribe` in `ngOnDestroy`.

Reply:
If we use `tap` instead of `subscribe` should be ok i guess. Please take a look on my proposal below.